content/exporters/supported-exporters/Go/ocagent.md

```diff
@@ -170,6 +170,14 @@ import (
 )
 
 func main() {
+	zPagesMux := http.NewServeMux()
+	zpages.Handle(zPagesMux, "/debug")
```

Comment:
We forgot to import zpages in the import block.

Reply:
Good catch, fixed.